emergence_lib/src/graphics/ui/mod.rs

```diff
@@ -0,0 +1,12 @@
+//! Creates the UI from all modules.
+//!
+// use bevy::prelude::Plugin;
```

Comment:
I am fine with us uncommenting the stuff in this file

Reply:
I figured since there was no plugin to build at the moment it would be left as a framework for later additions. I planned to add more fleshed out code with the pheromone UI but wanted it there for others to build on if they got to UI elements before me.

Reply:
@wallamide sure! totally up to you, just letting you know that it's also fine (and sometimes preferable) to leave stubs instead of commenting them out